### PR TITLE
[16.0][ADD] report_qweb_decimal_place

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
@qrtl

### CODE REVIEW

**Review by yostashiro — APPROVED** [review]
Code review.

**Comment by yostashiro** [comment]
> This migration depend on this PR (#697)

I think we should treat this PR as an addition instead of migration.

File: report_qweb_decimal_place/reports/price_unit_value_format.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="price_unit_value_format">
+        <span
+            class="text-nowrap"
+            t-esc="price_unit"
+            t-options='{"widget": "float", "precision": currency.price_decimal_places}'
+        />
```
Comment (yostashiro) [inline report_qweb_decimal_place/reports/price_unit_value_format.xml]:
```suggestion
        <t t-if="currency.apply_price_decimal_place">
            <span
                class="text-nowrap"
                t-esc="price_unit"
                t-options='{"widget": "float", "precision": currency.price_decimal_places}'
            />
        </t>
```
As we talked earlier, to minimize the code in the extension of individual reports.

File: report_qweb_decimal_place/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <attribute
+                name="t-att-style"
+            >'display: none' if doc.currency_id.apply_price_decimal_place else ''</attribute>
+        </xpath>
+        <xpath expr="//span[@t-field='line.price_unit']" position="after">
+            <t t-if="doc.currency_id.apply_price_decimal_place">
+                <t t-set="currency" t-value="doc.currency_id" />
+                <t t-set="price_unit" t-value="line.price_unit" />
+                <t t-call="report_qweb_decimal_place.price_unit_value_format" />
+            </t>
```
Comment (yostashiro) [inline report_qweb_decimal_place/readme/DESCRIPTION.rst]:
```suggestion
            <t t-set="currency" t-value="doc.currency_id" />
            <t t-set="price_unit" t-value="line.price_unit" />
            <t t-call="report_qweb_decimal_place.price_unit_value_format" />
```

**Review by yostashiro — APPROVED** [review]
👍

**Review by kanda999 — APPROVED** [review]

**Pull request merged**